Documentation/known-issues.md

```diff
@@ -0,0 +1,61 @@
+Known issues & workarounds
+==========================
+
+This document outlines the known issues and workarounds for the current state of 
+the CLI tools. Issues will also have a workaround and affects sections if necessary. You can use this page to 
+get information and get unblocked. 
+
+This page lists only the major issues that block users in doing their everyday tasks and that affect all or 
+most of the commands in the CLI tools. For minor issues, your best bet is still to check out the issues in 
+this repo. 
+
+# Current issues (12/23/2015)
+
+## El Capitan support
+El Capitan's RID is not supported properly currently. 
```

Comment:
I'd not include RID here. it's an implementation detail folks can learn about from GitHub issues. Let's keep this short & sweet: problem/solution.